igneous/tasks/skeletonization/skeletonization.py

```diff
@@ -61,11 +61,13 @@ def TEASAR(
   labels = np.asfortranarray(labels)
   DBF = np.asfortranarray(DBF)
 
+  soma_mode = dbf_max > soma_detection_threshold
   # > 5000 nm, gonna be a soma or blood vessel
   # For somata: specially handle the root by 
   # placing it at the approximate center of the soma
-  if dbf_max > soma_detection_threshold:
+  if soma_mode:
     root = np.unravel_index(np.argmax(DBF), DBF.shape)
+    soma_radius = (DBF[root[0],root[1],root[2]]*soma_invalidation_scale)+soma_invalidation_const
```

Comment:
Is this just: `soma_radius = dbf_max * soma_invalidation_scale + soma_invalidation_const`?

Reply:
yes.. and fixed